src/database/migrations/20210104113400-course-audit.ts

```diff
@@ -0,0 +1,36 @@
+import { DataTypes } from 'sequelize';
+import { QueryInterface } from 'sequelize';
+
+export default {
+    up: async (queryInterface: QueryInterface): Promise<void> => {
+        // Transactions are automatically use because a namespace is injected into sequelize when fetching configurations
+        await queryInterface.sequelize.transaction(async () => {
+            await queryInterface.addColumn('course_topic_content', 'course_topic_content_errors', {
```

Comment:
I think it's fine, but adding `count` to the end would make it much more clear what this is doing